index.html

```diff
@@ -11,35 +11,40 @@
   <body>
     <header>
       <h1 id="logo">Aziz Mejri</h1>
-      <button type="button" class="menu-bars">
+      <button type="button" id="open-menu-mobile" onclick="openMenuMobile()">
         <i class="fas fa-bars"></i>
       </button>
+
+      <nav id="mobile-nav">
+        <button type="button" id="close-menu-mobile" onclick="closeMenuMobile()">
+          <i class="fas fa-times"></i>
+        </button>
+        <ul class="mobile-nav-link-list">
+          <li><a class="mobile-nav-link" href="#projects">Portoflio</a></li>
+          <li><a class="mobile-nav-link" href="#about-me">About</a></li>
+          <li>
+            <a class="mobile-nav-link" href="#contact">
+              Contact <i class="far fa-envelope"></i>
+            </a>
```

Comment:
- [ ] As the previous reviewer already mentioned, when the user clicks any of the menu options the menu should disappear. this is due to the fact that there is no onclick event assigned to the button which should close the menu. please add the event handler similar to how you did in line 14, in order to add the close function to your application.